src/Uno.UWP/ApplicationModel/Calls/PhoneCallHistoryEntryReader.Android.cs

```diff
@@ -0,0 +1,140 @@
+﻿#nullable enable
+
+#if __ANDROID__

```

Comment:
I think this is not necessary. The file is already `.Android.cs` and is only included in build for Android.